content/develop/connect/clients/php.md

```diff
@@ -0,0 +1,314 @@
+---
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: Connect your PHP application to a Redis database
+linkTitle: PHP
+title: PHP guide
+weight: 6
+---
+
+[`Predis`](https://github.com/predis/predis) is the recommended [PHP](https://php.net/)
+client for Redis. 
+The sections below explain how to install `predis` and connect your application to a Redis database.
+
+{{< note >}}Although we provide basic documentation for `Predis`, it is a third-party
+client library and is not developed or supported directly by Redis.
+{{< /note >}}
+
+`Predis` requires a running Redis or
+[Redis Stack]({{< relref "/operate/oss_and_stack/install/install-stack/" >}}) server.
+See [Getting started]({{< relref "/operate/oss_and_stack/install/" >}}) for Redis installation
+instructions.
+
+## Install
+
+Use [Composer](https://getcomposer.org/) to install the `predis` library
```

Comment:
Choose one capitalization of predis and use throughout, unless there's a reason it's capitalized in some places, but not others.

```suggestion
Use [Composer](https://getcomposer.org/) to install the `Predis` library
```